tests/unit/core/TestInstallAddon.js

```diff
@@ -231,7 +231,7 @@ describe('withInstallHelpers inner functions', () => {
         guid,
         installURL,
       });
-      const setCurrentStatus = root.prop('setCurrentStatus');
+      const { setCurrentStatus } = root.instance().props;
```

Comment:
Ahhhhhhhhh. Oops. Okay, so `root.props()` for most components will continue to work as it does now, this was just a bit off in the past.

Cool, thanks!